test/sql/merge/merge_into_parenthesis_bug.test

```diff
@@ -0,0 +1,37 @@
+# name: test/sql/merge/merge_into_parenthesis_bug.test
+# description: Test MERGE issue when condition is in parenthesis
+# group: [merge]
+
+statement ok
+PRAGMA enable_verification
+
+statement ok
+CREATE TABLE my_timeseries (ts TIMESTAMP, x DOUBLE PRECISION, y DOUBLE PRECISION);
+
+statement ok
+insert into my_timeseries VALUES ('2025-09-15', 43, 39)
```

Comment:
Probably a stupid question but should all of the statements in the tests end with `;` It seems to be optional since this one doesn't have it.

Would it be better for the style to include them here too?

Reply:
I don't think that's a bad idea. Maybe the formatter could check and add these if necessary. I'm happy to review a PR if you want to have a go :-).